src/review/routes/Error.js

```diff
@@ -14,31 +14,18 @@ const styleSheet = createStyleSheet('Error', () => ({
   },
 }))
 
-function ErrorView({ classes, error }) {
-  return (
-    <ViewContainer>
-      <ReviewAppBar />
-      <ScrollView>
-        <LayoutBody margin bottom={false}>
-          { <div className={classes.statusCode}>Error {error.statusCode}</div> }
-          { error.message && <div>{error.message}</div> }
-          { error.stack && <code><pre>{error.stack}</pre></code> }
-        </LayoutBody>
-      </ScrollView>
-    </ViewContainer>
-  )
-}
-
-ErrorView.propTypes = {
-  classes: PropTypes.object,
-  error: PropTypes.shape({
-    message: PropTypes.string,
-    stack: PropTypes.string,
-    statusCode: PropTypes.number.isRequired,
-  }),
-}
-
 export default compose(
   withStyles(styleSheet),
   connect(state => ({ error: state.data.error })),
-)(ErrorView)
+)(({ classes, error }) => (
```

Comment:
- we don't care about displayName, did you already use it since Argos dev has started?
- consistency is respected for me:

´´´js
export default 'bar'
// I have to decorate it
export default transform('bar')
´´´

The consistency you mean is to write:

´´´js
const foo = 'foo'
export default foo
// I have to decorate it
const foo = 'foo'
export default transform(foo)
´´´

The consistence is not respected for me, you just declare a useless variable.